### PR TITLE
adding volumes and volume mounts to jupyter and worker configs

### PR DESCRIPTION
**Pull request by lbrindze**
Adds volume and volume mount configs to dask helm template. #76

### CODE REVIEW

**Review by jminsk-cc — APPROVED** [review]

**Comment by darothen** [comment]
Hey @TomAugspurger and @jacobtomlinson, we were working on some dask integration details for an internal project and wanted to share the fruits of our labor with the broader community. We didn't see a contributor's guide here, can you guys help us review and (hopefully!) merge this new feature in?

**Comment by lbrindze** [comment]
Also fwiw we (@jminsk-cc, @darothen and myself, as well as other members of the climacell team), are more than happy to help contribute in other helm related PRs since we are beginning to make this a key part of our data platform moving forward.

**Review by jminsk-cc — COMMENTED** [review]
More than happy to work on any other problems. Ran into the issue of not having an easy way to mount volumes onto workers. Accidentally left a review.

File: dask/values.yaml [hunk 1]
```diff
@@ -69,6 +69,13 @@ worker:
   #    cpu: 1
   #    memory: 3G
   #    nvidia.com/gpu: 1
+  mounts: {}
```
Comment (jacobtomlinson) [inline dask/values.yaml:72]: Could we add a description comment here?

```suggestion
  mounts: {}  # Describe me!
```

File: dask/values.yaml [hunk 2]
```diff
@@ -117,6 +125,13 @@ jupyter:
   #  requests:
   #    cpu: 2
   #    memory: 6G
+  mounts: {}
```
Comment (jacobtomlinson) [inline dask/values.yaml:128]:
```suggestion
  mounts: {}  # And describe me too
```

**Review by jacobtomlinson — COMMENTED** [review]
This looks great. Thanks Climacell team!

You may also want to be aware of #78 

Just a couple of comments, could you add descriptions to the new config options and regenerate the README?

**Comment by lbrindze** [comment]
@jacobtomlinson Sounds good I will update this shortly.  I just took a look at #78 and didn't see it before. Do we have a preference here in terms of passing the existing kubernetes yaml api through vs specifically overriding specific variables and passing them one at a time.   As this overlaps some with #78 I want to make sure we are only going forward with 1 solution here so any recommendations on how to best proceed here?

**Comment by jacobtomlinson** [comment]
@lbrindze I see no reason why we couldn't do both.

My general preference is to pass the k8s YAML API through, but having some bespoke options to simplify things for folks new to k8s is also nice to have.

**Comment by lbrindze** [comment]
@jacobtomlinson updated with the readme comments so if you think both options are good and will be ok to support moving forward I think this is ready to merge (pending any other feedback of course).

**Comment by jacobtomlinson** [comment]
Very happy to see this thanks @lbrindze

**Pull request merged**